lib/match.js

```diff
@@ -27,6 +27,7 @@ class Match extends Has {
       : pattern === String ? typeof obj === 'string'
       : pattern === Symbol ? typeof obj === 'symbol'
       : pattern === Boolean ? typeof obj === 'boolean'
+      : pattern === global.BigInt ? typeof obj === 'bigint'
```

Comment:
`BigInt` doesn't exist in node.js 8, referencing as a property of `global` works in 10 - 13 without breaking node.js 8.  This change is tested in node.js 12 only since the snapshots seem to be different for every version of node.js.